.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+
+# Pull Request
+
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+### Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
+## How Has This Been Tested?
+
+Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
+
+### Checklist
+
+- [ ] `terraform fmt` and `terraform validate` both work from the root and `examples/` directories (look in CI for an example)
+- [ ] Tests for the changes have been added and passing (for bug fixes/features)
```

Comment:
Is this relevant? We only test for formatting and validation, we don't have acceptance tests right now.

Reply:
I will re-phrase that to clarify that we want the changes implemented in an example